jfrog-cli/artifactory/commands/generic/upload.go

```diff
@@ -176,3 +163,41 @@ type UploadConfiguration struct {
 	ArtDetails            *config.ArtifactoryDetails
 	Retries               int
 }
+
+func GetUploadParams(f *spec.File, configuration *UploadConfiguration) (uploadParams services.UploadParams, err error) {
```

Comment:
This function can be made private.
Also, the function includes a lot of new-lines, which make the code a bit less convenient to read. This applies to a few other functions as part of this PR. Please consider modifying them as well.